pages/en/lb3/Discovering-models-from-relational-databases.md

```diff
@@ -243,3 +243,92 @@ ds.discoverExportedForeignKeys('PRODUCT',  cb);
 // Create a model definition by discovering the given table
 ds.discoverSchema(table, {owner: 'STRONGLOOP'}, cb);
 ```
+### Options
+
+We also provide several options for discovering models, they can be set by `options.*`:
```

Comment:
what do you mean by

> `options.*` 

please provide an example.

Do we mean that people can set up the options object like this?

```
const options = {relations: true, another option, another option};
  // Discover models and relations
  const inventorySchemas = await db.discoverSchemas('INVENTORY', options);
```

Reply:
yes, edited.